docs/hadoop-catalog.md

```diff
@@ -9,9 +9,9 @@ license: "This software is licensed under the Apache License version 2."
 ## Introduction
 
 Hadoop catalog is a fileset catalog that using Hadoop Compatible File System (HCFS) to manage
-the storage location of the fileset. Currently, it supports local filesystem and HDFS. For
-object storage like S3, GCS, Azure Blob Storage and OSS, you can put the hadoop object store jar like
-`gravitino-aws-bundle-{gravitino-version}.jar` into the `$GRAVITINO_HOME/catalogs/hadoop/libs` directory to enable the support.
+the storage location of the fileset. Currently, it supports the local filesystem and HDFS. Since 0.7.0-incubating, Gravitino supports [S3](hadoop-catalog-with-S3.md), [GCS](hadoop-catalog-with-gcs.md), [OSS](hadoop-catalog-with-oss.md) and [Azure Blob Storage](hadoop-catalog-with-adls.md) through Hadoop catalog. 
+
+The rest of this document will use HDFS or local file as an example to illustrate how to use the Hadoop catalog. For S3, GCS, OSS and Azure Blob Storage, the configuration is similar to HDFS, please refer to the corresponding document for more details.
```

Comment:
> The rest of this document will use HDFS or local file as an example to illustrate how to use the Hadoop catalog.

but you still list the configuration of S3, GCS, OSS and Azure Blob Storage below

Reply:
Do you suggest that we can remove those configurations from the document and only add them to cloud-storage-specific documents?

Reply:
Let me evaluate the affection and do corresponding steps